### ISSUE TITLE
[Android] Accessibility text scaling below 100% does not work (maybe intermittent)

### ISSUE THREAD

**Issue by timchilds** [brave/brave-browser#34233]
## Description
The Accessibility text scaling feature works when increasing text above 100%. But if you decrease text scaling below 100% it does not seem to do anything (the text size doesn't change). Maybe an intermittent issue?

## Steps to reproduce 

   1. Navigate to a website. e.g. techcrunch.com
   2. Go to settings -> accessibility
   3. Change the text scaling option to below 100%
   4. Go back to the website from (1). Try refreshing.

## Actual result
Text size on website does not get smaller.

## Expected result
Text size on website reduced to % scale chosen in step 3 above.

## Issue reproduces how often
Easily reproduced

## Version/Channel Information:
Release/1.60

## Device details
Phone / Android 12
   
## Brave version
Brave 1.60.110, Chromium 119.0.6045.105

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? Yes

### Additional information

https://github.com/brave/brave-browser/assets/28269287/05a6c4c3-9f9c-4ca6-aa73-e8154e45046b

- https://appfigures.com/reviews/266342430751LLjg38FQ1Goy5FpyYrKJP-g?lang=en
- https://appfigures.com/reviews/266342430751LvdvFUdRv_FRxLr5Bf9vUzQ?lang=en
- https://appfigures.com/reviews/266342430751LEUuhnBlxzvdWnnHVr6utSw?lang=en
- https://appfigures.com/reviews/266342430751LzHEDN8dGPJSufrXk0f1zxQ?lang=en
- https://appfigures.com/reviews/266342430751L5xMSrb47frgsqlTpln0xNw?lang=en
- https://appfigures.com/reviews/266342430751LffjgIr-ECyAvkBx3CA6AqQ?lang=en
